drs/drs-assembly/subscribe-model.conf

```diff
@@ -593,19 +671,33 @@ Current position of the cold stop. Contains the following:
 
 {
   subsystem =  IRIS
-  component =  ifs-scale-assembly.scaleMechanism
-  name =  position
+  component =  ifs-scale-assembly
+  name =  scaleMechanism
```

Comment:
Yes, I think Greg had this because he was working with older version of scale assembly publishing model file. I'll change appropriately.